public/javascripts/SVValidate/src/label/LabelVisibilityControl.js

```diff
@@ -32,6 +32,8 @@ function LabelVisibilityControl () {
     function unhideLabel () {
         let panomarker = svv.panorama.getPanomarker();
         let label = svv.panorama.getCurrentLabel();
+        var marker = document.getElementById('validate-pano-marker');
+        marker.classList.add('icon-outline');
```

Comment:
I think that you can probably replace these two lines with just
```
panomarker.marker_.classList.add('icon-outline');
```

We're already getting the panomarker here, so it's just about getting the actual HTML element out of it.

Reply:
If you do end up needing to save a variable, try to use `let` instead of `var` if possible!

Reply:
>I tried to combine the two lines in `LabelVisibilityControl.js` as you suggested but it didn't work the way I thought it would, so I just changed `var` to `let`

Interesting... I just tested it out locally while using your branch and it worked exactly the same! In fact, if I log both of these to the console, I get identical output:
```
console.log(document.getElementById('validate-pano-marker'));
console.log(panomarker.marker_);
```